uni/ulsp/jsonrpc-test.icn

```diff
@@ -0,0 +1,107 @@
+import ulsp
+
+global work_items
+
+#############################
+# JRPC server Calculator
+#############################
+class JRPC_Calc(jrpc_server)
+
+   method  start(addr)
+      local sock, msg
+      if not (jrpc_server := JRPC_HTTPSocket(addr,"n")) then
+         write("server open(",addr,") ERROR: ", \&errortext | "Unknown") & fail
+
+      repeat {
+         msg := jrpc_server.get_msg() | next
+         case msg.get_kind() of {
+            "request" : thread handle_request(msg)
+            default:
+               write("jrpc:msg: " || msg.get_kind(), image(msg.get_content()))
+         }
+      }
+   end
+
+   method handle_request(msg)
+      local id := msg.get_id(),
+            x := integer(msg.get_params()[1]),
+            y := integer(msg.get_params()[2])
+
+      # uncomment to handle potentially trigger some test failures
```

Comment:
There is meaning struggling to escape from this comment, but I can't quite work out what it is :-)